samples/Common.c

```diff
@@ -584,6 +585,15 @@ STATUS createSampleConfiguration(PCHAR channelName, SIGNALING_CHANNEL_ROLE_TYPE
 
     CHK_STATUS(lookForSslCert(&pSampleConfiguration));
 
+    // Set the logger log level
```

Comment:
I am a little confused. When we set the `AWS_KVS_LOG_LEVEL` environment variable, is it not the log level that is used for the entire SDK? 

Also, do we really want to jump to CleanUp section if log level setting fails? There is a default log level the SDK operates at even if the env is not set, correct?

Reply:
Yes, the logging level is used by the default logger in PIC. The failure is only for parsing. If the env variable is not set we default to warn level which is the default anyway

Reply:
Apologies for my verbatim. I meant parsing in my above comment. But do we really need to return from the function if the parsing fails? Is it not enough to log that the parsing failed?

Reply:
Hmm, perhaps you are right. Even when parsed we shouldn't fail. Let me modify the code really quick.